src/effects/builtin/autopaneffect.cpp

```diff
@@ -101,7 +101,7 @@ void AutoPanEffect::processChannel(
     AutoPanGroupState& gs = *pGroupState;
     double width = m_pWidthParameter->value();
     double period = m_pPeriodParameter->value();
-    double smoothing = 0.5 - m_pSmoothingParameter->value();
+    float smoothing = static_cast<float>(0.5 - m_pSmoothingParameter->value());
```

Comment:
General: We should use `auto` on the left-hand side if the actual type explicitly appears on the right-hand side to reduce redundancy.

Reply:
+1